src/components/portal/portal.scss

```diff
@@ -0,0 +1,5 @@
+.carbon-portal {
+  z-index: 1002;
+  position: absolute;
+  top: 0;
```

Comment:
how come we need the CSS for this? we didn't need it for datepicker?

Reply:
DatePicker does have css in ../component/date/date.scss 
Suppose we could do this input-validation css but shouldn't we generalise this for the Portal. Would we ever need the Portal anywhere else than position 0x0

Reply:
that's a good point, maybe we could generalise this a bit. Does that mean we could remove that CSS from datepicker too?

Might be good to use the `z-index` sass file https://github.com/Sage/carbon/blob/master/src/style-config/z-index.scss